src/content/docs/usage/pulse.md

```diff
@@ -23,3 +23,17 @@ With Bulk Restore, you can:
 * **Multi-Selection Mode:** Easily activate multi-select mode, choose the tests and suites you wish to restore, and bring them all back with a single action.
 
 ![Testomat.io - Bulk Restore](./images/New_85QaXaUI_2024-07-30.gif)
+
+## Trash Bin
```

Comment:
I think it is better to move that part to the end of https://docs.testomat.io/getting-started/test-design/ and call it `How to restore deleted tests`.

As people look for solutions not for feature names

However we can mention Pulse there as it allows to review all changes on project, including deletions